### PR TITLE
Reenable Span.CopyToLargeSizeTest

### PR DESCRIPTION
**Pull request by AtsushiKan**
Fixes https://github.com/dotnet/corefx/issues/25254

Let's see if using memory-mapped files makes this test
less punishing.

### CODE REVIEW

**Comment by AtsushiKan** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build

**Comment by AtsushiKan** [comment]
@dotnet-bot test CROSS Check

File: src/System.Memory/tests/Span/CopyTo.cs [hunk 1]
```diff
@@ -192,12 +195,17 @@ public static void CopyToCovariantArray()
...
+            public string TestFilePath => GetTestFilePath(0, "SpanTests_CopyToLargeSizeTest");
+        }
+
         // This test case tests the Span.CopyTo method for large buffers of size 4GB or more. In the fast path,
         // the CopyTo method performs copy in chunks of size 4GB (uint.MaxValue) with final iteration copying
         // the residual chunk of size (bufferSize % 4GB). The inputs sizes to this method, 4GB and 4GB+256B,
         // test the two size selection paths in CoptyTo method - memory size that is multiple of 4GB or,
         // a multiple of 4GB + some more size.
-        [ActiveIssue(25254)]
+
```
Comment (ahsonkhan) [inline src/System.Memory/tests/Span/CopyTo.cs]: nit: remove extra space

File: src/System.Memory/tests/ReadOnlySpan/CopyTo.cs [hunk 2]
```diff
@@ -123,12 +126,16 @@ public static void Overlapping2()
             Assert.Equal<int>(expected, a);
         }
 
+        private sealed class CopyToLargeSizeTestFileCleanup : FileCleanupTestBase
+        {
+            public string TestFilePath => GetTestFilePath(0, "ReadOnlySpanTests_CopyToLargeSizeTest");
```
Comment (ahsonkhan) [inline src/System.Memory/tests/ReadOnlySpan/CopyTo.cs]: Do we need two separate test files for the test? One for Span and for ReadOnlySpan?
Similarly, can we have just one implementation of the CopyToLargeSizeTestFileCleanup class?

**Comment by ahsonkhan** [comment]
For Windows.10.Nano.Amd64.Open-Debug-x64 -  System.SpanTests.ReadOnlySpanTests/CopyToLargeSizeTest(bufferSize: 4294967296)
https://mc.dot.net/#/user/AtsushiKan/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/b6059c448b0ae6d2cb5b171aaf69b59b281e9a89/workItem/System.Memory.Tests/analysis/xunit/System.SpanTests.ReadOnlySpanTests~2FCopyToLargeSizeTest(bufferSize:%204294967296)

```text
Unhandled Exception of Type System.IO.IOException
Message :
System.IO.IOException : There is not enough space on the disk
Stack Trace :
   at System.IO.FileStream.SetLengthCore(Int64 value) in E:\A\_work\1522\s\src\mscorlib\shared\System\IO\FileStream.Windows.cs:line 398
   at System.SpanTests.ReadOnlySpanTests.CopyToLargeSizeTest(Int64 bufferSize) in D:\j\workspace\windows-TGrou---db113413\src\System.Memory\tests\ReadOnlySpan\CopyTo.cs:line 159
```

**Comment by AtsushiKan** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build

File: src/System.Memory/tests/ReadOnlySpan/CopyTo.cs (hunk 2 above)
Comment (AtsushiKan) [reply src/System.Memory/tests/ReadOnlySpan/CopyTo.cs]: The string is just salt for the name - the infrastructure rolls in a Guid to create a new file regardless.

**Comment by AtsushiKan** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build

**Pull request closed without merging**